flang/lib/Optimizer/CodeGen/Target.cpp

```diff
@@ -205,6 +205,55 @@ struct TargetAArch64 : public GenericTarget<TargetAArch64> {
 };
 } // namespace
 
+//===----------------------------------------------------------------------===//
+// PPC64le (PowerPC 64 bit little endean) linux target specifics.
+//===----------------------------------------------------------------------===//
+
+namespace {
+struct TargetPPC64le : public GenericTarget<TargetAArch64> {
```

Comment:
Should this be GenericTarget<TargetAArch64> or GenericTarget<TargetPPC64le>?

Reply:
Ah yes, my bad. Fixed that